internal/resources/tanzupackageinstall/spec/cluster_scope.go

```diff
@@ -119,13 +135,16 @@ func FlattenSpecForClusterScope(spec *packageinstallmodel.VmwareTanzuManageV1alp
 	pkgRefSpec[PackageMetadataNameKey] = pkgMetadataName
 	pkgRefSpec[VersionSelectionKey] = versionSelectionSpec
 
+	// To be deprecated in a future release.
```

Comment:
same comment, Why can't we remove this code, and once this change is released this will not be supported.

Reply:
Users might not be able to update to this new schema in a short span of time so we are following the generic API deprecation flow which we use for TMC as well.